src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -4,15 +4,22 @@ import { NavigationMixin } from 'lightning/navigation';
 import messageLoading from '@salesforce/label/c.labelMessageLoading';
 import geSave from '@salesforce/label/c.labelGeSave';
 import geCancel from '@salesforce/label/c.labelGeCancel';
+import geDonationTypeErrorLabel from '@salesforce/label/c.geErrorDonorTypeValidation';
 import geUpdate from '@salesforce/label/c.labelGeUpdate';
 import { showToast, handleError, getRecordFieldNames, setRecordValuesOnTemplate } from 'c/utilTemplateBuilder';
 import { getQueryParameters, isNotEmpty } from 'c/utilCommon';
 import { getRecord } from 'lightning/uiRecordApi';
+import { format, isEmpty } from 'c/utilCommon';
 import FORM_TEMPLATE_FIELD from '@salesforce/schema/DataImportBatch__c.Form_Template__c';
 import TEMPLATE_JSON_FIELD from '@salesforce/schema/Form_Template__c.Template_JSON__c';
 import STATUS_FIELD from '@salesforce/schema/DataImport__c.Status__c';
-import NPSP_DATA_IMPORT_BATCH_FIELD
-    from '@salesforce/schema/DataImport__c.NPSP_Data_Import_Batch__c';
+import NPSP_DATA_IMPORT_BATCH_FIELD from '@salesforce/schema/DataImport__c.NPSP_Data_Import_Batch__c';
+// Import required schema for donation type validation
+import ACCOUNT1_NAME_FIELD_INFO from '@salesforce/schema/DataImport__c.Account1_Name__c';
+import ACCOUNT1_IMPORTED_FIELD_INFO from '@salesforce/schema/DataImport__c.Account1Imported__c';
```

Comment:
These lines are already used in utilTemplateBuilder.js, can we get them from there?

Reply:
there are 2 lines that are not included: 
```
import CONTACT1_LASTNAME_FIELD_INFO from '@salesforce/schema/DataImport__c.Contact1_Lastname__c';
import CONTACT1_LASTNAME_FIELD_INFO from '@salesforce/schema/DataImport__c.Account1_Name';
```
Should i move these ones as well? Will name them accordingly to what they're being called in utilTemplateBuilder